las/client.py

```diff
@@ -1021,9 +1022,22 @@ def create_prediction(
         :param auto_rotate: Whether or not to let the API try different rotations on\
             the document when running predictions
         :type auto_rotate: bool, optional
-        :param image_quality: image quality for prediction "LOW|HIGH". \
+        :param image_quality: Image quality for prediction "LOW|HIGH". \
             high quality could give better result but will also take longer time.
         :type image_quality: int, optional
+        :param postprocess_config: Post processing configuration for prediction.
+            {
+                'strategy': 'BEST_FIRST' | 'BEST_N_PAGES',  (required)
+                'parameters': {                             (required if strategy=BEST_N_PAGES)
+                    'n': int,                               (required if strategy=BEST_N_PAGES)
+                    'collapse': True | False                (optional if strategy=BEST_N_PAGES)
+                }
+            }
+            Examples:
+            {'strategy': 'BEST_FIRST'}
+            {'strategy': 'BEST_N_PAGES', 'parameters': {'n': 3}}
+            {'strategy': 'BEST_N_PAGES', 'parameters': {'n': 3, 'collapse': False}}
```

Comment:
I dont think this will look nice in the docs, examples should be put outside of the params list.